bad.js

```diff
@@ -1,3 +1,8 @@
-window.addEventListener('popstate', () => {
-  console.log('Doing something here!')
-})
\ No newline at end of file
+module.exports = {
+  sum: (a, b) => {
+    return a + b
+  },
+  addThing: () => {
+    window.addEventListener('popstate', () => {})
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/masterqwerty/sca-testing/pullRequest?prid=7714369)
```suggestion
    window.addEventListener("popstate", () => {})
```